Contents/mods/CommunityAPI/media/lua/client/WorldSoundAPI/WorldSoundAPIClient.lua

```diff
@@ -74,7 +74,7 @@ local Sounds = {}
 ---@param x number
 ---@param y number
 ---@param z number
----@param soundList string|table<string>
+---@param soundList string|table<number, string>
```

Comment:
It would be preferable to create a new branch for changing other API in case there is an other branch already working on it.

Reply:
Reverted param fix